static/js/lineChart.js

```diff
@@ -0,0 +1,305 @@
+function addAxesAndLegend (svg, xAxis, yAxis, margin, chartWidth, chartHeight) {
```

Comment:
This is unreadable. You should include it as a library as-is, and then use it, or if you need to modify it, fork it.